packages/loader/container-loader/src/connectionStateHandler.ts

```diff
@@ -59,6 +59,8 @@ export interface IConnectionStateHandler {
 	initProtocol(protocol: IProtocolHandler): void;
 	receivedConnectEvent(details: IConnectionDetailsInternal): void;
 	receivedDisconnectEvent(reason: string, error?: IAnyDriverError): void;
+	establishingConnection(reason: string): void;
+	disconnect(reason: string): void;
```

Comment:
I think it's weird to have receivedDisconnectEvent() & disconnect(). Maybe they are different thing, but there is no documentation in here to help the user understand it.

Reply:
Changed the name which is self explanatory and also added comment.